include/rmm/mr/device/tracking_resource_adaptor.hpp

```diff
@@ -253,7 +251,7 @@ class tracking_resource_adaptor final : public device_memory_resource {
           // Don't throw but log an error. Throwing in a descructor (or any noexcept) will call
           // std::terminate
           RMM_LOG_ERROR(
-            "Alloc bytes ({}) and Dealloc bytes ({}) do not match", allocated_bytes, bytes);
+            "Alloc bytes (%d) and Dealloc bytes (%d) do not match", allocated_bytes, bytes);
```

Comment:
*issue (perhaps?)*: throughout, `%d` is the format string for _signed integers_. But these are `std::size_t` values, so we need a `%uz` here

Reply:
Thanks. Note that in my reading, technically this should be `%zu` for broadest compatibility. All modifiers including the length modifier (`z`) go before the conversion format specifier (`u`).

Reply:
Removed all instances of %d for more appropriate formatters.